src/Schema/Events/Tap.ts

```diff
@@ -327,6 +327,50 @@ export interface TappedFairGroup extends TappedEntityGroup {
   action: ActionType.tappedFairGroup
 }
 
+/**
+ * A user taps a grouped explore module (City Guide, for example)
+ *
+ * This schema describes events sent to Segment from [[tappedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedCardsGroup",
+ *    context_module: "exploreBySection",
+ *    context_screen_owner_type: "home",
+ *    destination_screen_owner_type: "collectionsCategory",
+ *    destination_screen_owner_slug: "alex-katz-black-dress",
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface TappedCardsGroup extends TappedEntityGroup {
```

Comment:
We usually have the entity in the event name singular: so `tappedCardGroup`.

Reply:
changed it to singular in the latest commits 👍